diplom.py

```diff
@@ -9,6 +9,23 @@ def parse(line):
     label = int (tokens[23])
     return (features, label)
 
+def mean_average_precision (predicted_probability_distribution, 
```

Comment:
align arguments to parenthesis:
def func_name(argument_a,
                         argument_b,
                         argument_c):
not 
def func_name(argument_a,
   argument_b,
   argument_c):